web/src/HUD.test.tsx

```diff
@@ -71,7 +71,7 @@ it("doesn't re-render the sidebar when the logs change", async () => {
   let resourceView = oneResourceView()
   hud.setState({ View: resourceView })
   let oldDOMNode = root.find(".Sidebar").getDOMNode()
-  resourceView.Resources[0].PodLog += "hello world\n"
+  resourceView.Resources[0].ResourceInfo.PodLog += "hello world\n"
```

Comment:
This test was actually doing nothing because `PodLog` is on the ResourceInfo, not on the Resource 😭